sbi/utils/test_utils.py

```diff
@@ -0,0 +1,95 @@
+import torch
+import torch.distributions
+import sbi.utils as utils
+from sbi.simulators.linear_gaussian import (
+    get_true_posterior_log_prob_linear_gaussian_n_prior,
+    get_true_posterior_log_prob_linear_gaussian_mvn_prior,
+)
+from sbi.inference.posteriors.sbi_posterior import Posterior
+
+
+def dkl_gaussian_prior(
```

Comment:
should have a `test_` in front, no?

Reply:
Thanks! This (and the one below) are not tests in the sense that they should automatically be run by pytest. Instead, these functions are called inside a pytest function. To make this clearer, I renamed the file from `test_utils.py` to `utils_for_testing.py`. Does that make sense?

Reply:
I see, but they look very much like a test in the sense that they get the posterior and then assert an max dkl.